pkgs/by-name/hy/hydra/package.nix

```diff
@@ -123,7 +123,7 @@ let
         UUID4Tiny
         XMLSimple
         YAML
-        (nix.libs.nix-perl-bindings or nix.perl-bindings)
+        (nix.libs.nix-perl-bindings or nix.perl-bindings or null)
```

Comment:
I don't like this, but making it throw here breaks evaluation.  I haven't really wrapped my head around why that is, when aliases typically throw.  Is it because it's throwing during instantiation and not during the meta check?